src/diffusers/utils/export_utils.py

```diff
@@ -138,3 +140,162 @@ def export_to_video(
         img = cv2.cvtColor(video_frames[i], cv2.COLOR_RGB2BGR)
         video_writer.write(img)
     return output_video_path
+
+
+def colormap(
```

Comment:
Can be safely removed from the pipeline code then I guess?

Reply:
Yes it is already removed from the pipeline, now it lives in export_utils